### DISCUSSION TITLE
Support Time Travel of Arctic Mixed Hive Table

### DISCUSSION THREAD

**Discussion by wangtaohz** [apache/amoro#833] [General]

**Comment by zhoujinsong** [comment]
Time travel is a very interesting feature for me.
But before we start to support it, I think it is necessary to figure out how we can use this feature in production scenarios.

As far as I know, users have the need to view data at a certain point in time.
But it seems difficult to do this by time travel alone, you also need to generate a data snapshot at this point in time.
In addition, when we usually specify the time, we may want to specify a certain business timestamp column, not the data ingestion time.
